spec/grape/request_spec.rb

```diff
@@ -62,6 +62,24 @@ module Grape
       end
     end
 
+    describe 'when the param_builder is set to Hashie' do
+      before do
```

Comment:
I think this is overcomplicating things and stubs it to a point where it can actually be buggy and we wouldn't notice. Always better to do the real thing! Try this:

```ruby
before do
   Grape.configure do |config|
     config.praram_builder = Grape::Extensions::Hashie::Mash::ParamBuilder
   end
end

after do
  Grape::Config.reset
end
```

I actually think the `reset` could belong in spec/support/config.rb or something like that and be invoked every spec to make sure any global configuration is cleared for future use.